numpyro/distributions/continuous.py

```diff
@@ -1033,6 +1033,115 @@ def icdf(self, q):
         return self.loc + self.scale * logit(q)
 
 
+def _kron(A, B):
+    D = A[..., :, None, :, None] * B[..., None, :, None, :]
+    ds = D.shape
+    newshape = (*ds[:-4], ds[-4] * ds[-3], ds[-2] * ds[-1])
+    return D.reshape(newshape)
+
+
+class MatrixNormal(Distribution):
+    arg_constraints = {
+        "loc": constraints.real_vector,
+        "scale_columns": constraints.positive_definite,
+        "scale_rows": constraints.positive_definite,
+    }
+    support = constraints.real_matrix
+    reparametrized_params = ["loc", "scale_columns", "scale_rows"]
+
+    def __init__(self, loc, scale_columns, scale_rows, validate_args=None):
```

Comment:
I think it is better to use `scale_tril_row`, `scale_tril_column`, which has `constraints.lower_cholesky` constraint.